packages/boot/test/fast-usdc/snapshots/fast-usdc.test.ts.md

```diff
@@ -15,16 +15,8 @@ Generated by [AVA](https://avajs.dev).
       [
         'published.fastUsdc.feedPolicy',
         {
-          chainPolicies: {
-            Arbitrum: {
-              attenuatedCttpBridgeAddress: '0xe298b93ffB5eA1FB628e0C0D55A43aeaC268e347',
-              cctpTokenMessengerAddress: '0x19330d10D9Cc8751218eaf51E8885D058642E08A',
-              chainId: 42161,
-              confirmations: 2,
-            },
-          },
-          nobleAgoricChannelId: 'channel-21',
-          nobleDomainId: 4,
+          body: '#{"chainPolicies":{"Arbitrum":{"attenuatedCttpBridgeAddress":"0xe298b93ffB5eA1FB628e0C0D55A43aeaC268e347","blockWindowSize":10,"cctpTokenMessengerAddress":"0x19330d10D9Cc8751218eaf51E8885D058642E08A","chainId":42161,"confirmations":2,"maxAmountPerBlockWindow":"+20000000000"}},"nobleAgoricChannelId":"channel-21","nobleDomainId":4}',
```

Comment:
the function for documenting vstorage supports rendering this nicely. There's a show function or something.